CONTRIBUTION_GUIDELINES.md

```diff
@@ -0,0 +1,37 @@
+# Contribution Guidelines
+
+We welcome your contributions to help improve this package. Your efforts are greatly appreciated in making this project better for everyone. To contribute, please follow these guidelines:
+
+1. **Fork the Repository**: Begin by forking the repository and then clone it to your local machine.
+
+    ```
+    git clone https://github.com/your-username/repository-name.git
```

Comment:
can you update this url to proper url `https://github.com/user-focus/marvin-utils.git`